### PR TITLE
ENCD-5664-juicebox-availability-fix

### PR DESCRIPTION
**Pull request by yunhailuo**

### CODE REVIEW

**Review by gabdank — APPROVED** [review]

File: src/encoded/vis_defines.py [hunk 1]
```diff
@@ -1716,10 +1718,7 @@ def browsers_available(
                 and status in VISIBLE_FILE_STATUSES):
             # use of find_or_make_acc_composite() will recurse!
             vis_blob = vis_cache.get(accession=accession, assembly=assembly)
-        if not vis_blob and file_assemblies is None and files is not None:
```
Comment (hitz) [inline src/encoded/vis_defines.py:1719]: @yunhailuo why did you remove the "if not vis_blob" clause?   OK I see it now.

**Review by hitz — APPROVED** [review]
